### PR TITLE
ZKUI-174: Cold storage location improvement

### PR DESCRIPTION
**Pull request by ChengYanJin**
Bump with the latest core-ui version v0.38.3

Location listing:
image
Location creation:
image
Location editing:
image

### CODE REVIEW

File: src/react/backend/location/LocationDetails/LocationDetailsHyperdriveV2.tsx [hunk 1]
```diff
@@ -34,13 +34,18 @@ export default class LocationDetailsHyperdriveV2 extends React.Component<
...
             id="bootstrapList"
-            entries={this.state.bootstrapList}
-            listLimit={MAX_HYPERDRIVE}
-            onUpdate={this.onListChange}
+            required
+            label="Bootstrap List"
+            getInputProps={() => ({
+              autoComplete: 'off',
+              type: 'text',
+              placeholder: 'example:localhost:8181',
```
Comment (JBWatenbergScality) [inline src/react/backend/location/LocationDetails/LocationDetailsHyperdriveV2.tsx]:
```suggestion
placeholder: 'example: localhost:8181',
```

File: src/react/backend/location/LocationDetails/LocationDetailsSproxyd.tsx [hunk 2]
```diff
@@ -77,18 +77,18 @@ export default class LocationDetailsSproxyd extends React.Component<
...
             id="bootstrapList"
-            entries={this.state.bootstrapList}
-            listLimit={SPROXYD_LIMIT}
-            onUpdate={this.onListChange}
+            required
+            label="Bootstrap List"
+            getInputProps={() => ({
+              autoComplete: 'off',
+              type: 'text',
+              placeholder: 'example:localhost:8181',
```
Comment (JBWatenbergScality) [inline src/react/backend/location/LocationDetails/LocationDetailsSproxyd.tsx]:
```suggestion
              placeholder: 'example: localhost:8181',
```

File: src/react/backend/location/LocationDetails/LocationDetailsSproxyd.tsx [hunk 3]
```diff
@@ -92,7 +92,9 @@ export default class LocationDetailsSproxyd extends React.Component<
           />
         </Fieldset>
         <Fieldset>
-          <Label htmlFor="">Proxy Path</Label>
+          <Label htmlFor="" required>
```
Comment (JBWatenbergScality) [inline src/react/backend/location/LocationDetails/LocationDetailsSproxyd.tsx]:
```suggestion
          <Label htmlFor="proxyPath" required>
```

File: src/react/ui-elements/Input.ts [hunk 4]
```diff
@@ -42,15 +42,20 @@ const Input = styled.input`
...
     outline: none;
     border: ${spacing.sp1} solid ${(props) => props.theme.brand?.secondary};
   }
-
+  ${(props) =>
+    props.disabled &&
+    `  pointer-events: auto;
+cursor: not-allowed;
+opacity: 0.5;
+`}
```
Comment (JBWatenbergScality) [inline src/react/ui-elements/Input.ts:58]: Ideally this should be in core ui. Also @Cuervino I'm wondering if similarly as select component we should add the disabled icon at the extreme right of the input

Comment (ChengYanJin) [reply src/react/ui-elements/Input.ts:58]: I will create a debt ticket to migrate the input component with the one we have in core-ui.

**Review by JBWatenbergScality — APPROVED** [review]

**Pull request merged**